portfolio/src/main/java/com/google/sps/servlets/UserComment.java

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+public final class UserComment {
```

Comment:
Should this file and related classes be called UserLocations now? Feel free to just TODO that as well, I know this PR has been hanging around a while.

Reply:
I think UserComment is most appropriate for this class because it stores more than just location. It stores name, phone #, location, and description so calling it UserLocation or something along those lines would be confusing.